content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,136 @@
+---
+reviewers:
+- verb
+- yujuhong
+title: Debug a Running Pod
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+This page shows how to investigate problems in a running pod using a
+debugging ephemeral container. This is useful when the container being
```

Comment:
Is Ephemeral Container a k8s resource?
nit: line 13, using an Ephemeral Container

Reply:
The deadline is Monday Sep 9.